app/javascript/packs/pages/provider_signup/SignupStepper.jsx

```diff
@@ -108,10 +132,12 @@ export default function SignupStepper({ steps }) {
     setActiveStep(activeStep - 1);
   };
 
-  const onChange = (e) => {
-    setVariables({...variables, [e.target.name]: e.target.value});
+  const setField = name => value => {
```

Comment:
Is there a reason you're doing composed functions here instead of

```
const setField = (name, value) => {
```
?

Reply:
I tend to default to currying in JS for the flexibility of partial application but I didn't actually take advantage of that here. Happy to change it if you'd like.